cmd/explaintest/r/new_character_set_invalid.result

```diff
@@ -21,3 +21,33 @@ a	b	c
 ?	?	?
 中文?中文	asdf?fdsa	字符集?字符集
 @@	@@	@@
+drop table t;
+create table t(f set(0xD2BB, 0xC8FD), e enum(0xBAEC,0x6A59));
+show create table t;
+Table	Create Table
+t	CREATE TABLE `t` (
+  `f` set('һ','??') DEFAULT NULL,
+  `e` enum('??','jY') DEFAULT NULL
```

Comment:
MySQL only have one '?', do we need to compatible with it?

Reply:
MariaDB also has 2 question marks... Anyway, let's do it when it is necessary.